reactjs/src/scenes/Roles/index.tsx

```diff
@@ -159,6 +172,15 @@ class Role extends AppComponentBase<IRoleProps> {
           >
             <Button type="primary" shape="circle" icon="plus" onClick={() => this.createOrUpdateModalOpen({ id: 0 })} />
           </Col>
+        </Row>        
+        <Row>
+          <Col sm={{ span: 5, offset: 0 }}>
```

Comment:
AntDesign has Search Component. You could use the component as follows: 
const Search = Input.Search;

<Search
      placeholder={L('SearchPlaceHolder')}
      onSearch={this.handleSearch}
      enterButton
    />